pages/learn/getting-started-step-by-step.md

```diff
@@ -363,68 +354,63 @@ To create a nested data structure:
 
 1. Inside the `properties` object, create a new key called `dimensions`:
 
-
-```jsonc
-...
-  "properties": {
-  ...
-    "dimensions": {}
-  }
-```
+    ```jsonc
+    ...
+      "properties": {
+      ...
+        "dimensions": {}
+      }
+    ```
 
 2. Define the `type` validation keyword as `object`:
 
-
-```jsonc
-...
-  "dimensions": {
-    "type": "object",
-  }
+    ```jsonc
+    ...
+      "dimensions": {
+        "type": "object",
+      }
 ```
 
 3. Add the `properties` validation keyword to contain the nested data structure. Inside the new `properties` keyword, add keywords for `length`, `width`, and `height` that all use the `number` type:
 
-
-```jsonc
-...
-  "dimensions": {
-    "type": "object",
-    "properties": {
-      "length": {
-        "type": "number"
-      },
-      "width": {
-        "type": "number"
-      },
-      "height": {
-        "type": "number"
+    ```jsonc
+    ...
+      "dimensions": {
+        "type": "object",
+        "properties": {
+          "length": {
+            "type": "number"
+          },
+          "width": {
+            "type": "number"
+          },
+          "height": {
+            "type": "number"
+          }
+        }
       }
-    }
-  }
 ```
```

Comment:
here too

Reply:
Huge thanks for finding this!

Bhavuk can you create a new PR to fix this? I merged it too fast.